### PR TITLE
Remove layout styling abstraction

### PR DESCRIPTION
**Pull request by Velin92**

### CODE REVIEW

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineStyler.swift [hunk 1]
```diff
@@ -20,8 +20,6 @@ import SwiftUI
 // MARK: - TimelineStyler
 
 struct TimelineStyler<Content: View>: View {
```
Comment (Velin92) [inline ElementX/Sources/Screens/RoomScreen/View/Style/TimelineStyler.swift:22]: @pixlwave  Maybe we could rename this to `TimelineStylerContainer` and the `TimelineBubbleStylerView` to `TimelineStyler`

Comment (pixlwave) [reply ElementX/Sources/Screens/RoomScreen/View/Style/TimelineStyler.swift:22]: No strong feelings, although it's nice to type `bubble` when searching for the part of the code where message bubbles are made.

**Review by pixlwave — APPROVED** [review]
Thank you!

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineStyler.swift (hunk 1 above)
Comment (Velin92) [reply ElementX/Sources/Screens/RoomScreen/View/Style/TimelineStyler.swift:22]: then let's leave it as it is

**Pull request merged**